### DISCUSSION TITLE
Updating manual installation

### DISCUSSION THREAD

**Discussion by madcoda9000** [photoview/photoview#517] [Q&A and Troubleshooting] [answered]
Hello,

at first, THIS IS A SEXY PEACE OF SOFTWARE!!!! Many thanks for that. I've followed your guide for the manual setup (because i dont want to use docker) with reverse Proxy and it is working like a charm.

One question left: how do i make an update? 

Greetings
Sascha

**Comment by viktorstrate** [comment, marked as answer]
You repeat the steps from `Download and build Photoview` to `Copy needed files` but downloading the updated files. As long as the database doesn't change and the photo directories and cache directories doesn't change, then everything should be fine.

https://photoview.github.io/docs/installation-manual/#download-and-build-photoview

---
And btw, thank you very much for the kind words. I don't think anyone has called software i've written _sexy_ before, especially not in ALL CAPS. ❤️
